tests/chordsymbol_tests.ts

```diff
@@ -8,14 +8,18 @@ import { TestOptions, VexFlowTests } from './vexflow_test_helpers';
 import { Accidental } from '../src/accidental';
 import { ChordSymbol } from '../src/chordsymbol';
 import { Factory } from '../src/factory';
+import { Font, FontGlyph } from '../src/font';
```

Comment:
The changes in this file were also probably added by mistake.

Reply:
Maybe I missing something?  Maybe you are a PR behind?  I see the code here:

https://github.com/0xfe/vexflow/blob/master/tests/chordsymbol_tests.ts

It only shows up here because I did a merge to my vexflow repository.